shiny/ui/_sidebar.py

```diff
@@ -79,14 +83,27 @@ class SidebarOpen:
     )
 
     @staticmethod
-    def _values_str() -> str:
-        return f"""'{"', '".join(SidebarOpen._VALUES)}'"""
+    def _values_str(values: tuple[str, ...] | None = None) -> str:
+        quoted = [f'"{x}"' for x in (values or SidebarOpen._VALUES)]
+        ret = ""
+        for i, value in enumerate(quoted):
+            if i > 0 and len(quoted) > 2:
+                ret += ","
+            if i == len(quoted) - 1:
+                ret += " or"
+            if i > 0:
+                ret += " "
+            ret += value
+        return ret
```

Comment:
This formatting logic is not worth it IMO. And actually, the default `__str__` method on tuples does something pretty sensible:

```python
>>> x = ("foo", "bar")
>>> f"Must be one of these: {x}"
"Must be one of these: ('foo', 'bar')"
``` 

Motivated by this and a few other things that bothered me (the existence of `_always_open_mobile`, etc) I came up with this diff:

[sidebar-changes.patch](https://github.com/user-attachments/files/16241353/sidebar-changes.patch)

Reply:
Thanks, these were great suggestions. I applied them with small edits: 0e01c66faf7e87d5092635dd416faf1069720863